tasks/33_array-plus-array/array-plus-array.js

```diff
@@ -0,0 +1,24 @@
+
+
+export function arrayPlusArray(arr1, arr2) {
+    
+
+    if (arr1.length >= arr2.length) {
+       
+
+    for (let i = 0; i<arr1.length; i++) {
+        if (i < arr2.length) arr1[i] += arr2[i];
+    }
+
+    return arr1;
+    
+
+    } else {
+        for (let i = 0; i<arr2.length; i++) {
+            if (i < arr1.length) arr2[i] += arr1[i];
+        }
+        return arr2;
+    }
```

Comment:
for loops are repeated and it would be nice to extract them into separate function so it can be called:

`export function arrayPlusArray(arr1, arr2) {
    arr1.length >= arr2.length ?
      return addA(arr1, arr2) :
      return addA(arr2, arr1);
}

function addA(arr1, arr2) {
  for (let i = 0; i < arr1.length; i++) {
    if (i < arr2.length) arr1[i] += arr2[i];
  }

  return arr1;
}`